test/e2e/utils/config.go

```diff
@@ -26,18 +26,47 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	DefaultConfigPath             = "/var/lib/kubelet/config.yaml"
+	DefaultPodResourcesSocketPath = "/var/lib/kubelet/pod-resources/kubelet.sock"
+)
+
 var (
 	e2eConfigFile = flag.String("nfd.e2e-config", "", "Configuration parameters for end-to-end tests")
 
 	config *E2EConfig
 )
 
+type KubeletConfig struct {
+	ConfigPath             string
+	PodResourcesSocketPath string
+}
+
 type E2EConfig struct {
 	DefaultFeatures *struct {
 		LabelWhitelist      lookupMap
 		AnnotationWhitelist lookupMap
 		Nodes               []NodeConfig
 	}
+
+	Kubelet *KubeletConfig
+}
+
+func (conf *E2EConfig) GetKubeletConfig() KubeletConfig {
```

Comment:
nit: missing "docstring" 🙄 Just caught my eye when skimming through. I know we are missing these in many places of the existing code but trying to improve things slowly...

Reply:
no problem, fixed!